extensions/wikia/TemplateClassification/UnusedTemplates/UnusedTemplatesHandler.class.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace Wikia\TemplateClassification\UnusedTemplates;
+
+class Handler {
+	/**
+	 * Reflects a `propname` in the `page_wikia_props` table.
+	 */
+	const PROPERTY_NAME = 11000;
```

Comment:
It's a property name according to how it's called in the table (look at `page_wikia_props` schema)